src/cpp/server/server_cc.cc

```diff
@@ -587,6 +590,7 @@ class Server::CallbackRequest final : public Server::CallbackRequestBase {
   const bool has_request_payload_;
   grpc_byte_buffer* request_payload_;
   void* request_;
+  void* handler_data_ = nullptr;
```

Comment:
Any reason for a default initializer like this?

Reply:
Removed, as Setup will be called in the ctor to cover this anyways.